datafusion/expr/src/built_in_function.rs

```diff
@@ -710,30 +704,6 @@ impl BuiltinScalarFunction {
             BuiltinScalarFunction::Digest => {
                 utf8_or_binary_to_binary_type(&input_expr_types[0], "digest")
             }
-            BuiltinScalarFunction::Encode => Ok(match input_expr_types[0] {
```

Comment:
This metadata information about the functions is now moved into the `functions/encoding/meta` module